integration-tests/environments/electron/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "realm-electron-tests",
+  "description": "Test harness running the Realm JS tests in the Electron renderer process.",
+  "version": "0.1.0",
```

Comment:
Nitpick, but wouldn't `1.0` be a better choice or if completely irrelevant `0.0.0`?

Reply:
I was just thinking of https://stackoverflow.com/a/24154727/503899 .. I think its fine :)